### ISSUE TITLE
[BUG] Assess & Refactor NPC Generator

### ISSUE THREAD

**Issue by msprijatelj** [msprijatelj/4e-module-generator-portable-compendium#4]
**Describe the bug**
The NPC generator does not capture some information normally, like Recharge values, Descriptions, and Equipment.  It also has cases where some enemies lack their complete stat blocks, usually ones with Auras that are not formatted as traits (like many early 4e statblocks), sometimes to an egregious degree (see Orcus).

**Steps to Reproduce**
1. Run `npc.exe` to generate a Feat module.
2. Load up Fantasy Grounds with that module.
3. Check "Naarash Warcaster" under NPCs
4. See that the Recharge values are missing numbers.
5. Check "Lamia" under NPCs
6. See that the "Swarm's Embrace" aura is missing altogether.
7. Check "Orcus" under NPCs
8. See that most of Orcus's statblock is missing, save for the Wand of Orcus attack.

**Expected behavior**
The generated NPCs include Recharge information, aura information for old statblocks, and Descriptions/Equipment in the NPC's "Other" tab on Fantasy Grounds (if they exist in the Compendium).

**Comment by EthanDunning** [comment]
Hey, I think the title for this is wrong, its Feat instead of NPC

**Comment by EthanDunning** [comment]
In messing around with the cobmat tracker today, I also noticed that creatures HP start out as 0 on the combat tracker. I believe this to be an error in the coding, as trying out my ddi parsed modules, they work fine.

**Comment by EthanDunning** [comment]
I have figured out what is causing it, working exported data has health as, for example, "60; Bloodied 30", and ours only has "60". will fix code and make a pull request soon.

**Comment by skelekon** [comment]
Yes I remember the HP 0 thing was a bug in the original VegaFontana importer, in that there was a space in front of all monster HP values.
I used a regex in Notepad++ to replace them all and they started showing up correctly.
